procedures/observing_tools/Sky Almanac.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>You can remove the commented line.</p><p>Maybe add a line above this that says something like, "Declare targets for which the sky coordinates will be calculated"</p><p> </p><p>Also, "Beginning of the night"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>